swift/test/Ice/udp/AllTests.swift

```diff
@@ -6,37 +6,25 @@ import Ice
 import TestCommon
 
 class PingReplyI: PingReply, @unchecked Sendable {
-    var _replies: Int32 = 0
-    var _lock = os_unfair_lock()
-    var _semaphore = DispatchSemaphore(value: 0)
+    private let _semaphore = DispatchSemaphore(value: 0)
```

Comment:
All the updates in this PR are straightforward except the updates to this file.

The main fix is to get rid of the _replies property and just use the DispatchSemaphore, including its count.